src/main/java/sysc4806/project24/mini_survey_monkey/controllers/CreateController.java

```diff
@@ -82,4 +95,12 @@ public String deleteQuestion(
         questionRepository.deleteById(questionID);
         return "redirect:/create/" + surveyID;
     }
+
+    @GetMapping("/survey/{surveyID}")
+    public String viewSurvey(@PathVariable("surveyID") int surveyID, Model model) {
+        Survey survey = surveyRepository.findById(surveyID);
+        model.addAttribute("survey", survey);
+
+        return "survey";
+    }
```

Comment:
**Would it make sense to factor this to a different controller?**

For instance, `surveyController`?

It might make understanding the system easier if our controllers reflect the pages they are controlling.

For instance:
* `loginController`
* `analyticsController`

Reply:
Also, could change endpoint to `summary`id`.

This is how Survey Monkey does it.

Reply:
Maybe call it `summaryController`.